cargo-concordium/src/templates/cis2-nft-lib.rs

```diff
@@ -0,0 +1,1057 @@
+//! A NFT smart contract example using the Concordium Token Standard CIS2.
+//!
+//! # Description
+//! An instance of this smart contract can contain a number of different token
+//! each identified by a token ID. A token is then globally identified by the
+//! contract address together with the token ID.
+//!
```

Comment:
It seems unsatisfactory to duplicate this code from the examples folder. That's hard to maintain.

I assume this is the same example as the one in rust-smart-contracts?

Could we instead add that repository as a submodule here? Is there some other way we could link to those files instead?